proxy-node-shared/src/main/java/uk/gov/ida/notification/exceptions/proxy/VerifyServiceProviderRequestException.java

```diff
@@ -1,14 +1,14 @@
 package uk.gov.ida.notification.exceptions.proxy;
 
+import org.slf4j.MDC;
+import uk.gov.ida.notification.shared.ProxyNodeMDCKey;
+
 import javax.ws.rs.WebApplicationException;
 
 public class VerifyServiceProviderRequestException extends WebApplicationException {
-    private final String sessionId;
 
     public VerifyServiceProviderRequestException(Throwable cause, String sessionId) {
         super(cause);
-        this.sessionId = sessionId;
+        MDC.put(ProxyNodeMDCKey.SESSION_ID.name(), sessionId);
```

Comment:
We will _try_ to make the `ProxyNodeLogger` static in a separate PR, and at that time we could replace the call to `MDC.put(...)` with a static call to `ProxyNodeLogger.setContext(...)`

Reply:
Agreed.